packages/react-accordion/Spec.md

```diff
@@ -0,0 +1,299 @@
+| ------------------ | ----------------------- | ------------- | ------------------------------------------------------------ |
+| expand (?)         | boolean                 | false         | Allows multiple panels to be expanded                        |
+| collapse (?)       | boolean                 | false         | Allows multiple panels to be collapsed                       |
+| expandIconPosition | Enum                    | "end"         | Position of the icon to indicate expansion                   |
+| heading            | ElementType             | "h3"          | Heading element for the panel                                |
+| onToggle           | (open: boolean) => void |               | Dispatched when it's state changes between opened and closed |
+
+By default, the Accordion must have one, and **ONLY** one (1..1), opened panel all the time.
+If `collapse` is used, then the Accordion can have one or zero (0..1) panels opened.
+If `expand` is used, then the Accordion can have one or more (1..n) panels opened.
+If both `expand` and `collapse` are used, then the Accordion doesn't have restrictions (0..n).
+
+### AccordionPanel
+
+| Prop name   | Type           | Details                                                                                                    |
+| ----------- | -------------- | ---------------------------------------------------------------------------------------------------------- |
+| title       | ShorthandValue | Label for representing a section of content that also serves as control for showing and hiding the content |
+| children    | ShorthandValue | The content with visibility controlled by the panel                                                        |
+| open        | boolean        | Controls the state of the panel                                                                            |
+| defaultOpen | boolean        | Default value for the state of the panel                                                                   |
+| expandIcon  | ShorthandValue | Icon to indicate expansion                                                                                 |
+| disabled    | boolean        | Disables opening/closing of panel                                                                          |
+
+## Sample code
+
+The below samples do not represent the definitive props of the final implemented component, but represent the ideal final implementations. Can be subject to change during the implementation phase.
+
+### Basic Accordion
+
+```tsx
+const accordion = (
+  <Accordion>
+    <AccordionPanel title="First Panel">
+      This is the content of the first Panel
+    </AccordionPanel>
+    <AccordionPanel title="Second Panel">
+      This is the content of the second Panel
+    </AccordionPanel>
+  <Accordion>
+)
+```
+
+Expected DOM output
+
+```html
+<div>
+  <h3>
+    <button aria-expanded="false" aria-controls="sect1" id="accordion1">
```

Comment:
`button` can have quite aggressive restrictions on what the children could be, does it make sense to default to div with appropriate role and event handlers ?